pages/[slug].tsx

```diff
@@ -1,28 +1,33 @@
-import type { ExtendedStory } from '@prezly/sdk/dist/types';
+import type { Category, ExtendedStory } from '@prezly/sdk/dist/types';
 import { GetServerSideProps, NextPage } from 'next';
 import { getPrezlyApi } from '@/utils/prezly';
 import Story from '@/modules/Story';
 import Layout from '@/components/Layout';
 
 type Props = {
     story: ExtendedStory;
+    categories: Category[];
 };
 
-const StoryPage: NextPage<Props> = ({ story }) => (
-    <Layout>
+const StoryPage: NextPage<Props> = ({ story, categories }) => (
+    <Layout categories={categories}>
         <Story story={story} />
     </Layout>
 );
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
     const api = getPrezlyApi(context.req);
-    const { slug } = context.params;
+    const { slug } = context.params as { slug: string };
```

Comment:
Assuming the 21 line is correct, that means that slug can be undefined. Can it?

```suggestion
    const { slug } = context.params as { slug?: string };
```